server/sonar-process-monitor/src/test/java/org/sonar/application/cluster/AppStateClusterImplTest.java

```diff
@@ -116,6 +121,41 @@ public void simulate_network_cluster() throws InterruptedException {
     }
   }
 
+  @Test
+  public void appstateclusterimpl_must_set_sonarqube_version() {
+    TestAppSettings settings = newClusterSettings();
+
+    try (AppStateClusterImpl appStateCluster = new AppStateClusterImpl(settings)) {
+      appStateCluster.registerSonarQubeVersion(getSonarqubeVersion());
```

Comment:
no need to use `getSonarqubeVersion()`. It should be replaced by a fake value like "1.0".